src/test/Makefile

```diff
@@ -69,8 +69,8 @@ OBJ_TESTS = \
 	obj_many_size_allocs\
 	obj_realloc\
 	obj_sync\
-	\
```

Comment:
please leave this line as is - it separates longer tests from shorter

Reply:
Done.